R/pkg/R/generics.R

```diff
@@ -385,6 +385,7 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 #' @return A SparkDataFrame.
 #' @rdname summarize
 #' @export
+#' @noRd
```

Comment:
what's the reason for the noRd here? I think it's fairly standard to "document" generic - even though it's not useful most of the time